@AH-64D Apache Longbow/Addons/fza_ah64_systems/functions/core/fn_coreVariables.sqf

```diff
@@ -0,0 +1,67 @@
+/* ----------------------------------------------------------------------------
+Function: fza_systems_fnc_variables
+
+Description:
+    ...
+
+Parameters:
+	_heli      - The helicopter to get information from [Unit].
+
+Returns:
+    ...
+
+Examples:
+    ...
+
+Author:
+	BradMick
+---------------------------------------------------------------------------- */
+params ["_heli"];
+#include "\fza_ah64_systems\headers\systems.hpp"
+
+//Switch states
+_heli setVariable ["fza_systems_battSwitchState",   "OFF"];
+
+//Electrical System
+//--Battery
+_heli setVariable ["fza_systems_battPower_pct",     1.0];
+private _battTime = SYS_BATT_TIMER * 60;
+_heli setVariable ["fza_systems_battTimer",         _battTime];
+//--Buses
+_heli setVariable ["fza_systems_battBusState",      "OFF"];
+_heli setVariable ["fza_systems_ACBusState",        "OFF"];
+_heli setVariable ["fza_systems_DCBusState",        "OFF"];
+//--Gen 1 and RTRU 1
+_heli setVariable ["fza_systems_gen1State",         "OFF"];
+_heli setVariable ["fza_systems_rtru1State",        "OFF"];
```

Comment:
For values that are meant to be just "ON" and "OFF", we should not use string variables, but instead a boolean value. For example `["fza_systems_gen1On", true]`. That way you can do

```sqf
if (_heli getVariable "fza_systems_gen1On") then {
```

Reply:
Fixed.

Reply:
You are not my friend. Changed everything over.